devicon.json

```
@@ -1300,6 +1300,22 @@
         },
         "color": "#f24e1e",
         "aliases": []
+    },
+	{
+        "name": "filezilla",
+        "tags": [
+            "ftp"
+        ],
+        "versions": {
+            "svg": [
+                "plain", "plain-wordmark"
+            ],
+            "font": [
+                "plain"
```

This looks great now! I also notice that you changes the `wordmark` version as well, which will make it font-friendly. I think we can add this in too.
```suggestion
                "plain",
                "plain-wordmark"
```